android/test-app-util.gradle

```diff
@@ -223,14 +223,24 @@ ext.getVersionName = {
 }
 
 ext.isFabricEnabled = { project ->
-    return isNewArchitectureEnabled(project) ||
-        (isPropertySet(project, "USE_FABRIC", "1") &&
-            getPackageVersionNumber("react-native", project.rootDir) >= 6800)
+    if (isNewArchitectureEnabled(project)) {
```

Comment:
btw I have mentioned it somewhere, but from 0.71 there are going to be new "togglers" for Fabric and TM: 
* https://github.com/facebook/react-native/pull/35091
* https://github.com/facebook/react-native/pull/35117
don't know if you want to add this separately when trying out 0.71 rc0